utils/jwt.py

```
@@ -28,16 +28,19 @@ def is_token_blacklisted(token: str) -> bool:
         pass
     return True  # 실패 시 안전하게 막음
 
-# 쿠키에서 토큰 추출
-def get_token_from_cookie(request: Request):
-    token = request.cookies.get("Access")
-    if not token:
-        raise HTTPException(status_code=401, detail="No token in cookie")
+# 헤더에서 토큰 추출
+def get_token_from_header(request: Request):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)